foreman-release-scl/tfm-ror51.repo

```diff
@@ -0,0 +1,11 @@
+# As defined by https://copr.fedorainfracloud.org/coprs/g/theforeman/tfm-ror51/repo/epel-7/group_theforeman-tfm-ror51-epel-7.repo
+[group_theforeman-tfm-ror51]
+name=Copr repo for tfm-ror51 owned by @theforeman
+baseurl=https://copr-be.cloud.fedoraproject.org/results/@theforeman/tfm-ror51/epel-7-$basearch/
+type=rpm-md
+skip_if_unavailable=True
+gpgcheck=1
+gpgkey=https://copr-be.cloud.fedoraproject.org/results/@theforeman/tfm-ror51/pubkey.gpg
```

Comment:
Would it be a good idea to ship the GPG key in the package?

Reply:
I suppose we can? Though, if the GPG key is unavailable, so too will the packages.